Formula/capnp.rb

```diff
@@ -24,10 +24,23 @@ class Capnp < Formula
   depends_on "cmake" => :build
 
   def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args
-      system "make", "install"
-    end
+    # Build static library
+    system "cmake", "-S", ".", "-B", "build_static",
+                    "-DBUILD_SHARED_LIBS=OFF",
+                    "-DCMAKE_POSITION_INDEPENDENT_CODE=ON",
+                    *std_cmake_args
+    system "cmake", "--build", "build_static"
+    system "cmake", "--install", "build_static"
+
+    # Build shared library
+    system "cmake", "-S", ".", "-B", "build_shared",
+                    "-DBUILD_SHARED_LIBS=ON",
+                    "-DCMAKE_POSITION_INDEPENDENT_CODE=ON",
+                    "-DCMAKE_INSTALL_RPATH=#{rpath}",
+                    *std_cmake_args
+    system "cmake", "--build", "build_shared"
+    lib.install buildpath.glob("build_static/src/capnp/*.{so,dylib}")
+    lib.install buildpath.glob("build_static/src/kj/*.{so,dylib}")
```

Comment:
Let's not do this. `cmake` relinks binaries at install time, and we don't want to skip that. We should only do `lib.install` for static libraries.

Reply:
annoying, I wanted the executable to use the static link. I'll switch around